### ISSUE TITLE
Infinite reboot loop

### ISSUE THREAD

**Issue by Karol-Aniszewski** [myspaghetti/macos-virtualbox#623]
Hi ;)
I'm trying to install macOS Mojave, but on first boot the VM falls into a loop.
Any advices?

Screenshots:
![image](https://user-images.githubusercontent.com/59614308/192718896-98ad5c17-b0e4-4b11-a8e9-6910fb837a85.png)

![image](https://user-images.githubusercontent.com/59614308/192532888-595fda47-1918-49ed-9517-b546bc772c9a.png)

Video:
https://user-images.githubusercontent.com/59614308/192534339-8e03475d-3c61-45a7-a8d8-3acf39594a58.mp4

troubleshoot.txt:
[macOS_troubleshoot.txt](https://github.com/myspaghetti/macos-virtualbox/files/9656065/macOS_troubleshoot.txt)

**Comment by myspaghetti** [comment]
Thanks for the detailed bug report!

The VirtualBox log shows that VT-x/AMD-V is not available so VirtualBox falls back to NEM, which is not supported by the script (I haven't verified, but I suspect NEM can't run macOS on VirtualBox). [Some Windows feature or third-party app is blocking VirtualBox from using VT-x/AMD-V.](https://github.com/myspaghetti/macos-virtualbox#virtualbox-native-execution-manager-nem) 

People have reported running the following command in an Administrator `cmd.exe` instance and rebooting twice solves the issue:
```
bcdedit /set hypervisorlaunchtype off 
```
But software other than Windows itself might be blocking VT-x/AMD-V. Worth a try, though.

**Comment by myspaghetti** [comment]
Thanks for the detailed bug report!

The VirtualBox log shows that VT-x/AMD-V is not available so VirtualBox falls back to NEM, which is not supported by the script (I haven't verified, but I suspect NEM can't run macOS on VirtualBox). [Some Windows feature or third-party app is blocking VirtualBox from using VT-x/AMD-V.](https://github.com/myspaghetti/macos-virtualbox#virtualbox-native-execution-manager-nem) 

People have reported running the following command in an Administrator `cmd.exe` instance and rebooting twice solves the issue:
```
bcdedit /set hypervisorlaunchtype off 
```
But software other than Windows itself might be blocking VT-x/AMD-V, or it might be disabled in UEFI/BIOS, or it might not be available on the hardware at all. Worth a try, though.

**Comment by Karol-Aniszewski** [comment]
Hi, thank you for your answer.
Anyway issues with Hyper-V was not a case (I have disabled it long before, because I was using another VMs).
As I followed the clues from this video: https://www.youtube.com/watch?v=Lq8J-vFqH7w&ab_channel=OnlineComputerTips
I changed --cpuidset and some other "extra data" for my VM (except the GetKeyFromRealSMC) and it started working.
I suspect that this "cpuid" was causing problems, because the same issue was resolved in this topic:
https://www.tonymacx86.com/threads/kernel-panic-and-reboot-while-installing-sierra-g4500-h110.207109/
